.github/dependabot.yml

```
@@ -0,0 +1,21 @@
+---
+version: 2
+updates:
+  # Enable version updates for python
+  - package-ecosystem: "python"
```

See https://docs.github.com/en/code-security/supply-chain-security/configuration-options-for-dependency-updates

```suggestion
  - package-ecosystem: "pip"
```